main/modes/gamepad/gamepad.c

```diff
@@ -340,6 +344,12 @@ void gamepadStart(gamepadType_t type)
     }
     else
     {
+        const uint8_t macBytes = 6; // This is part of the ESP API's design, and cannot be changed here
+        uint8_t mac[macBytes];
+        if (ESP_OK == esp_wifi_get_mac(WIFI_IF_STA, mac))
+        {
+            memcpy(&hid_string_descriptor[3], mac, macBytes);
+        }
```

Comment:
Is the serial expecting alphanumeric or ASCII values? because the MAC is going to be over the place.

Reply:
And the string being overwritten is "123456" (which is also the combination to my luggage.

Reply:
@Brycey92 to check if there are any collisions with two swadges with the same serial. If not, remove this.

Reply:
Removed, no conflicts between 2 swadges